documentation/adoc/metrics.adoc

```diff
@@ -1,149 +1,137 @@
-# Metrics
+== Metrics
 
 This section describes how it's possible to deploy a Prometheus server for scraping metrics from the Kafka cluster and showing them using a Grafana dashboard. The provided resources for deploying can't be considered production grade; they are used just as an example to show how it's possible to use the metrics feature provided in this project.
 
 If you are going to use `minikube` or `minishift` for your Apache Kafka deployment adding Prometheus and Grafana servers, it would be better to increase the default amount of memory available to the virtual machine (i.e. not using the default 2 GB of RAM but 4 GB instead).
```

Comment:
The documentation should be written in the passive form. E.g. instead of _If you are going to use `minikube` or `minishift` for your Apache Kafka deployment ..._ it should say something like _When using `minikube` or `minishift` for Apache Kafka deployment ..._

The passive form should be used everywhere in the document.